modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -188,6 +190,19 @@ def buildProject(Map options = [:]) {
       }
     }
 
+    if (options.publishingE2eTests == true) {
+      setEnvar("FULL_COMMIT_HASH", sh(
+        script: "git rev-parse HEAD",
+        returnStdout: true
+      )
+      stage("End-to-end tests") {
+        appCommitishName = env.PUBLISHING_E2E_TESTS_APP_PARAM ||
+          repoName.replace("-", "_").toUpperCase() + "_COMMITISH"
+        testBranch = env.PUBLISHING_E2E_TESTS_BRANCH || "master"
```

Comment:
we've [very recently](https://github.com/alphagov/publishing-api/commit/835956e8fc22a23723b4814d8b693a82f4580c76) switched to using "test-against" as the default branch